### PR TITLE
[eclipse/xtext-core#1626] regnerate languages 

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext-core#1626] regnerate languages

### CODE REVIEW

File: org.eclipse.xtext.extras.tests/src-gen/org/eclipse/xtext/enumrules/services/EnumRulesTestLanguageGrammarAccess.java [hunk 1]
```diff
@@ -95,28 +96,27 @@
 		private final Keyword cDifferentNameDifferentLiteralKeyword_2_0 = (Keyword)cDifferentNameEnumLiteralDeclaration_2.eContents().get(0);
 		
 		//enum ExistingEnum:
-		//	SameName | OverriddenLiteral="overridden" | DifferentName="DifferentLiteral";
+		//  SameName | OverriddenLiteral = "overridden" | DifferentName
```
Comment (ArneDeutsch) [inline org.eclipse.xtext.extras.tests/src-gen/org/eclipse/xtext/enumrules/services/EnumRulesTestLanguageGrammarAccess.java:99]: Interesting ... the old logic using the serializer infers the value and generates an assignment, that is not present in the grammar. Of course the new logic is not capable of that.

File: org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/services/XbaseGrammarAccess.java [hunk 2]
```diff
@@ -96,21 +99,23 @@
...
+		//    =>({XBinaryOperation.leftOperand=current} feature=[types::JvmIdentifiableElement|OpMultiAssign]) rightOperand=XAssignment
+		//)?
 		public Group getGroup_1() { return cGroup_1; }
 		
 		//XOrExpression
 		public RuleCall getXOrExpressionParserRuleCall_1_0() { return cXOrExpressionParserRuleCall_1_0; }
 		
-		//(=> ({XBinaryOperation.leftOperand=current} feature=[types::JvmIdentifiableElement|OpMultiAssign])
-		//rightOperand=XAssignment)?
+		//(
```
Comment (ArneDeutsch) [inline org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/services/XbaseGrammarAccess.java:110]: Here is one of the rare cases that works not so well. But I would consider it minor.

**Review by ArneDeutsch — APPROVED** [review]

File: org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/services/XbaseGrammarAccess.java (hunk 2 above)
Comment (szarnekow) [reply org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/services/XbaseGrammarAccess.java:110]: Agreed

**Review by szarnekow — APPROVED** [review]

File: org.eclipse.xtext.extras.tests/src-gen/org/eclipse/xtext/enumrules/services/EnumRulesTestLanguageGrammarAccess.java (hunk 1 above)
Comment (szarnekow) [reply org.eclipse.xtext.extras.tests/src-gen/org/eclipse/xtext/enumrules/services/EnumRulesTestLanguageGrammarAccess.java:99]: For completeness: The serializer operates on the model which is augmented by the Xtext linker. The linker will create objects like the return type ref or the string literal for enumerations if the short-hand notation is used.
The transient value service is later responsible for skipping these again - but it is not perfect: The string literal `DifferentLiteral` would be optional.

**Pull request merged**